### ISSUE TITLE
[Question] How can I install it on Linux?

### ISSUE THREAD

**Issue by Toadfield** [MrOtherGuy/fx-autoconfig#3]
The instructions say that I should put the files in the same folder where firefox.exe is located,but that isn't the case for Linux,cause there are no .exe files for Linux.

**Comment by MrOtherGuy** [comment]
I would guess it depends where your distro (and how you installed it) puts the binary and associated files. Firefox installed from Fedora Software store application ("Software" is the name of the application) ended up in `/usr/lib64/firefox/`

I have not tried on linux but I woul expect that this folder would be used as "the folder containing firefox.exe"

**Comment by Toadfield** [comment]
So do I also have to put the "chrome" folder in there?

**Comment by MrOtherGuy** [comment]
No, the chrome folder goes to your profile folder, which you should be able to find the same way as on Windows. So you can follow the [setting up profile](https://github.com/MrOtherGuy/fx-autoconfig#setting-up-profile) section for that.

**Comment by LithRakoon** [comment]
how would i install this incase of the firefox flatpak?

**Comment by MrOtherGuy** [comment]
I honestly have no clue, you'll need to experiment.
